pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "microjson"
-version = "0.1.7"
+version = "0.1.9"
```

Comment:
Is there a reason to increment patch version of microjson package from 0.1.7 - to 0.1.9 and skipping 0.1.8

Reply:
I discovered some bugs in 0.1.8 just after bumping the version. But no, I should take greater care not to skip version. :)